src/Avalonia.Base/Input/KeyGesture.cs

```diff
@@ -155,7 +155,11 @@ private static Key ParseKey(string key)
             if (s_keySynonyms.TryGetValue(key.ToLower(), out Key rv))
                 return rv;
 
+#if NETSTANDARD2_0
```

Comment:
Maybe we can add a small helper somewhere to handle such parsing? Just so we don’t need to sprinkle #if everywhere

Reply:
Done.